go-controller/pkg/libovsdbops/db_object_types.go

```diff
@@ -105,3 +108,23 @@ var ACLNetpolNode = newObjectIDsType(acl, NetpolNodeOwnerType, []ExternalIDKey{
 	// exact ip for management port, every node may have more than 1 management ip
 	IpKey,
 })
+
+// ACLNetworkPolicy define a unique index for every network policy ACL.
+// ingress/egress + NetworkPolicy[In/E]gressRule idx - defines given gressPolicy.
+// acls are created for every gp.portPolicies:
+// - for empty policy (no selectors and no ip blocks) - empty ACL
```

Comment:
I dont see this addressed

Reply:
I updated it to
`for empty policy (no selectors and no ip blocks) - empty ACL (see constructEmptyMatchString)`
I don't think diving into semantics makes sense here, because that file is only about ids. So I thought just saying where the logic resides would be good enough (especially since what empty acl is and what it means may change in the future)

Reply:
ok, I would have called it a matchAllIP ACL or something, but ok if you want to leave it

Reply:
sure I can rename the function, if we agree on the comment itself